infinispan-spring-boot-starter-remote/src/test/java/test/org/infinispan/spring/starter/remote/CustomConfigurerTest.java

```diff
@@ -0,0 +1,52 @@
+package test.org.infinispan.spring.starter.remote;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.spring.starter.remote.InfinispanRemoteAutoConfiguration;
+import org.infinispan.spring.starter.remote.InfinispanRemoteConfigurer;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DirtiesContext
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(
+    classes = {
+        CustomConfigurerTest.TestConfiguration.class,
+        InfinispanRemoteAutoConfiguration.class
+    },
+    properties = {
+        "spring.main.banner-mode=off",
+        "myServerList=localhost:6667"
+    }
+)
+public class CustomConfigurerTest {
```

Comment:
Perhaps we could come up with a better name here? Something like `CustomConfigurerWithPropertyInjectionTest`?